### DISCUSSION TITLE
new use cases and reviews

### DISCUSSION THREAD

**Discussion by jonathangoeke** [GoekeLab/sg-nex-data#17] [Announcements]
The page that lists reviews and articles that use the SG-NEx data was updated. You can view the list here: https://github.com/GoekeLab/sg-nex-data/blob/master/SGNEx_usecases.md

Please add additional entries if you use the data in your research, we are happy to hear how the data is used!

Jonathan

**Comment by spuronlee** [comment]
Hi, in the Pre-Release (v0.2)  I can not find the links to download the raw fast5 file for dRNA-seq data, it is not available now.
